pkg/hub/controllers/workerslicegwrecycler/utils.go

```diff
@@ -260,7 +271,7 @@ func (r *Reconciler) update_routing_table(e *fsm.Event) error {
 		err = retry.Do(func() error {
 			workerslicegwrecycler.Status.Client.Response = slicerouter_updated
 			return r.Status().Update(ctx, workerslicegwrecycler)
-		})
+		}, retry.Attempts(20))
```

Comment:
Why retry number is comparatively less here? If it is expected, add a comment here.